core/ide/src/main/java/org/alice/stageide/sceneeditor/StorytellingSceneEditor.java

```diff
@@ -463,6 +475,44 @@ public void setSelectedExpression(Expression expression) {
     }
   }
 
+  private void updateOrthographicCameraView(EntityImp targetImplementation) {
+    double targetHeight = targetImplementation.getAxisAlignedMinimumBoundingBox().getHeight();
+    AffineMatrix4x4 targetTransform = targetImplementation.getAbsoluteTransformation();
+    ClippedZPlane picturePlane = new ClippedZPlane();
+
+    AffineMatrix4x4 topTransform = AffineMatrix4x4.createIdentity();
+    topTransform.translation.x = targetTransform.translation.x;
+    topTransform.translation.y = 10;
+    topTransform.translation.z = targetTransform.translation.z;
+    topTransform.orientation.up.set(0, 0, 1);
+    topTransform.orientation.right.set(-1, 0, 0);
+    topTransform.orientation.backward.set(0, 1, 0);
+    this.topOrthoMarkerImp.setLocalTransformation(topTransform);
+    picturePlane.setCenter(0, 0);
+    picturePlane.setHeight(16);
+    this.topOrthoMarkerImp.setPicturePlane(picturePlane);
+
+    AffineMatrix4x4 sideTransform = AffineMatrix4x4.createIdentity();
+    sideTransform.translation.x = targetTransform.translation.x + 10.0f;
```

Comment:
Again, the constant 10 could instead be based on the width of the axis aligned box built at the beginning of the method.